libraries/testkit/sgaccel.py

```diff
@@ -16,6 +16,9 @@ def __init__(self, cluster_config, target):
         self.ip = target["ip"]
         self.url = "http://{}:4985".format(target["ip"])
         self.hostname = target["name"]
+        self.cluster_config = cluster_config
+        self.server_port = 8091
+        self.server_scheme = "http"
```

Comment:
When initializing the class, why not see a bool `server_ssl`. Then below in `.start()`, just set the port and scheme as local variables. That way you can reduce the amount of state on the class.